onnxruntime/core/session/standalone_op_invoker.cc

```diff
@@ -531,8 +531,7 @@ ORT_API_STATUS_IMPL(OrtApis::CopyKernelInfo, _In_ const OrtKernelInfo* info, _Ou
 
 ORT_API(void, OrtApis::ReleaseKernelInfo, _Frees_ptr_opt_ OrtKernelInfo* info_copy) {
   if (info_copy) {
-    auto kernel_info = reinterpret_cast<onnxruntime::OpKernelInfo*>(info_copy);
-    delete kernel_info;
```

Comment:
Is this getting flagged? We've a lot of 'delete' in the source code esp. in onnxruntime_c_api.cc. Using unique_ptr here doesn't quite make sense to me. Same comment for attr and kernel info.

Reply:
Yes this is flagged - "Avoid calling new and delete explicitly, use std::make_unique<T> instead (r.11)."

Reply:
Use this: https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/core/session/onnxruntime_c_api.cc#L70

Reply:
Done.